atomic_reactor/plugins/build_orchestrate_build.py

```diff
@@ -64,6 +68,47 @@ def override_build_kwarg(workflow, k, v):
     override_kwargs[k] = v
 
 
+class UnknownPlatformException(Exception):
+    """ No clusters could be found for a platform """
+
+
+class AllClustersFailedException(Exception):
+    """ Each cluster has reached max_cluster_fails """
+
+
+class ClusterRetryContexts(object):
+    def __init__(self, clusters, max_cluster_fails, find_cluster_retry_delay):
+        self.clusters = set(clusters)
+        self.retry_at = {}
+        self.failed = set()
+        self.fails = Counter()
+        self.max_cluster_fails = max_cluster_fails
+        self.delay = find_cluster_retry_delay
```

Comment:
We had comments about some of these attributes in a previous iteration. Can we bring those back please?
https://github.com/projectatomic/atomic-reactor/pull/841/commits/bfc011d0635d8a17b1d7ad0763d5f4924a20b8c4